magicctapipe/scripts/lst1_magic/semi_automatic_scripts/job_accounting.py

```diff
@@ -190,83 +193,121 @@ def main():
         for file in ins:
             if os.path.exists(f"{dir}/logs/{file}"):
                 list_dl0 = f"{dir}/logs/{file}"
+
         if list_dl0 != "":
             with open(list_dl0, "r") as fp:
                 this_todo = len(fp.readlines())
+        elif args.data_level in ["DL2", "DL3"]:
+            if timerange and args.data_level == "DL3":
+                files = []
+                for date in (
```

Comment:
I think those complications are also related to droping date from DL3 directory name